hw2/05-form.html

```diff
@@ -45,8 +48,8 @@ <h2 class="text-center"><strong>Exercise 05 - Form</strong></h1>
     <input type="checkbox" class="form-check-input" id="FullStack">
     <label class="form-check-label" for="FullStack">Full Stack Web Development</label>
   </div>
-  <label for="InputComment" class="form-label"><strong>Anything Else?</strong></label>
-  <textarea class="pb-cmnt-textarea"></textarea>
+  <label for="InputComment" class="form-label" ><strong>Anything Else?</strong></label>
+  <textarea class="pb-cmnt-textarea" id ="Anything"></textarea>
   <div class="btn-group">
     <a href="#" class="btn btn-primary my-2 mx-1" id="Submit">Submit</a>
```

Comment:
Instead of using links <a> tab for button it would be a good idea to add <button> or <input>. You can include submit and reset as the type attributes for these.

Reply:
interesting, yeah you could use it as the same button that way. This way just made the most sense to me